packages/core/src/utils/index.js

```diff
@@ -1,5 +1,7 @@
 import * as constantsData from './constants';
 import * as helperFunctions from './helper-functions';
+import * as behaviorClasses from './behaviors';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Explicitly import the specific member needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS2208)